### PR TITLE
add server url configuration to BoxCollector and BoxV2

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-5222?filter=-1)

### CODE REVIEW

File: Packs/Box/ReleaseNotes/3_1_19.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+#### Integrations
+
+##### Box v2
+
+- Added `url` field to integration configuration.
```
Comment (ShahafBenYakir) [inline Packs/Box/ReleaseNotes/3_1_19.md:6]:
```suggestion
- Added *Server URL* parameter to the integration configuration. 
```

File: Packs/Box/ReleaseNotes/3_1_19.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+#### Integrations
+
+##### Box v2
+
+- Added `url` field to integration configuration. 
+- Updated the Docker image to: *demisto/pyjwt3:1.0.0.52151*.
+
+##### Box Event Collector
+
+- Added `url` field to integration configuration.
```
Comment (ShahafBenYakir) [inline Packs/Box/ReleaseNotes/3_1_19.md:11]:
```suggestion
- Added *Server URL* parameter to the integration configuration. 
```

**Review by ShahafBenYakir — APPROVED** [review]
Looks great

**Comment by DinaMeylakh** [comment]
@ShahafBenYakir 
The only failure is validations for 
```
[ERROR]: Packs/Box/Integrations/BoxV2/BoxV2.yml: [IN116] - You've added required, the field is 'url'
[ERROR]: Packs/Box/Integrations/BoxEventsCollector/BoxEventsCollector.yml: [IN116] - You've added required, the field is 'url'
```
Which is the point of the issue, I believe in this case a force merge is needed.

**Pull request merged**